src/PrestaShopBundle/Form/Admin/Sell/Product/Category/ProductCategoryType.php

```diff
@@ -30,26 +30,22 @@
 
 use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
 use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
-use Symfony\Component\Form\Extension\Core\Type\RadioType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
 use Symfony\Component\Form\FormBuilderInterface;
 
 class ProductCategoryType extends TranslatorAwareType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
+            ->add('name', HiddenType::class)
```

Comment:
You could use `TextPreviewType` here (https://github.com/PrestaShop/PrestaShop/blob/507e1ee9e7bfe937a3b0d8b7edc399f0a1f08840/src/PrestaShopBundle/Form/Admin/Type/TextPreviewType.php)

To simplify the template, you can then use `form_row(form.name)` (or form_widget) instead of accessing the data via `form.vars.name`